pixart.js

```diff
@@ -1 +1,36 @@
+var brush = $(".brush");
+var button = document.getElementById("set-color");
+var input = document.getElementById("color-field");
+function changeBrushColor (inputColor) {
+  brush.css("background", inputColor);
+}
 
+
+//I got very stuck on the part where I needed to enter the input.value, so I refered to the
+//solutions, but I'm not sure what the function(event) part of this does.
+
+button.addEventListener("click", function(event){
+  event.preventDefault();
+  changeBrushColor(input.value);
+})
+
+
+
```

Comment:
when you get some time, after the project is finished, try writing in the for loop
👍 good effort overall, as always